### ISSUE TITLE
Unlock trophys huh? 

### ISSUE THREAD

**Issue by SilicaAndPina** [soarqin/vita-rcheat#2]
MMMHMHMMM WONDER WHERE YOU GOT THAT FROM >_<

nah... its ok. . . i guess

**Comment by soarqin** [comment]
Yup the idea is from TropHAX, but I did read the documentation of leaked official SDK, took the SceNpTrophy module definitions from it and rewrote the whole trophy unlocking codes.
Thank you for your great work for TropHAX.
It is so sorry that I forgot to mention that, will add a 'Thanks to' part in README asap.
